packages/application-shell/src/components/application-shell/application-shell.tsx

```diff
@@ -461,10 +459,8 @@ export const RestrictedApplication = <
                                   }
                                 </Route>
                                 {/* Project routes */}
-                                <Route
-                                  exact={true}
-                                  path="/"
-                                  render={() => {
```

Comment:
No more `render` or `component` prop, routes are rendered as children of `<Route>`